src/System.Private.ServiceModel/tools/SelfHostedWcfService/SelfHostedWCFService.csproj

```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SelfHostedWcfService</RootNamespace>
     <AssemblyName>SelfHostedWcfService</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Deliberate change to allow use of MemoryStream.TryGetBuffer.  The old self-hosted service was using TargetFrameworkVersion v4.6